proto/poktroll/migration/tx.proto

```diff
@@ -111,3 +113,24 @@ message MsgClaimMorseAccountResponse {
   int64 claimed_at_height = 3;
 }
 
+// TODO_UPNEXT(@bryanchriswhite, #1034):
```

Comment:
Any chance we can do the comments in this PR because it prevents me from understanding the plan.